docs/dev_env/Ubuntu_Development_Environment.md

```diff
@@ -0,0 +1,146 @@
+<a href="/readme.md"><img src="/docs/images/PlayEveryWareLogo.gif" alt="README.md" width="5%"/></a>
+
+# <div align="center">Configuring Ubuntu 18.04 for development</div>
+
+# Setting up your environment:
+
+This is the second part of the guide for using Hyper-V to set up a Linux environment on your Windows PC that can be utilized for testing and developing the EOS Plugin. This guide _can_ however be used on its own as a guide to setting up a new Ubuntu 18.04 operating system for making contributions to the EOS Plugin.
+
+## Setup your Linux Guest VM in Hyper-V
+
+If you want to set up your linux environment inside a virtual machine, and have not yet done so, follow [this](/docs/dev_env/HyperV_Linux_Guest_VM.md) guide first.
+
+## How to expand Hyper-V disk space for Ubuntu (if using Hyper-V)
+
+This is for after you have Ubuntu installed. It is recommended to set your disk space to an appropriate size before you install the OS.
```

Comment:
You should add instructions to the Hyper-V setup process to adjust drive size, so people don't end up needing this.

Reply:
Good call - just added those, and moved this section to the end as a safety.